bitnami/keycloak/templates/statefulset.yaml

```
@@ -0,0 +1,204 @@
+          {{- else if .Values.auth.tls.enabled }}
+          {{- $fullname := include "keycloak.fullname" . }}
+          command:
+            - /bin/bash
+            - -ec
+            - |
+              ID="${MY_POD_NAME#"{{ $fullname }}-"}"
+              if [[ -f "/certs/keycloak.truststore.jks" ]] && [[ -f "/certs/keycloak-${ID}.keystore.jks" ]]; then
+                  mkdir -p /opt/bitnami/keycloak/certs
+                  cp "/certs/keycloak.truststore.jks" "/opt/bitnami/keycloak/certs/keycloak.truststore.jks"
+                  cp "/certs/keycloak-${ID}.keystore.jks" "/opt/bitnami/keycloak/certs/keycloak.keystore.jks"
+              else
+                  echo "Couldn't find the expected Java Key Stores (JKS) files! They are mandatory when encryption via TLS is enabled."
+                  exit 1
+              fi
+              exec /opt/bitnami/scripts/schema-registry/entrypoint.sh /opt/bitnami/scripts/schema-registry/run.sh
+          {{- end }}
+          {{- if .Values.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.args "context" $) | nindent 12 }}
+          {{- end }}
+          env:
+            - name: MY_POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: KUBERNETES_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.namespace
+            - name: BITNAMI_DEBUG
+              value: {{ ternary "true" "false" .Values.image.debug | quote }}
+            {{- if .Values.extraEnvVars }}
+            {{- include "common.tplvalues.render" (dict "value" .Values.extraEnvVars "context" $) | nindent 12 }}
+            {{- end }}
+          envFrom:
+            - configMapRef:
+                name: {{ printf "%s-env-vars" (include "keycloak.fullname" .) }}
+            - secretRef:
+                name: {{ printf "%s-env-vars" (include "keycloak.fullname" .) }}
+            {{- if .Values.extraEnvVarsCM }}
+            - configMapRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.extraEnvVarsCM "context" $) }}
+            {{- end }}
+            {{- if .Values.extraEnvVarsSecret }}
+            - secretRef:
+                name: {{ include "common.tplvalues.render" (dict "value" .Values.extraEnvVarsSecret "context" $) }}
+            {{- end }}
+          {{- if .Values.resources }}
+          resources: {{- toYaml .Values.resources | nindent 12 }}
+          {{- end }}
+          ports:
+            - name: http
+              containerPort: {{ .Values.containerPorts.http }}
+              protocol: TCP
+            - name: https
+              containerPort: {{ .Values.containerPorts.https }}
+              protocol: TCP
+            - name: http-management
+              containerPort: 9990
```

```suggestion
              containerPort: {{ .Values.metrics.service.port }}
```

Probably a typo. A related question; should the port `http-management` be inside a `{{- if .Values.metrics.enabled }}`?

---

It's not a typo, please note that "containerPort != servicePort". For instance, I can use **9990** as container port, and **8888** in the service.